packages/devtools/kit/src/rsdoctor.ts

```diff
@@ -0,0 +1,37 @@
+import type { Manifest } from '@rsdoctor/types';
+
+export async function fetchShardingFile(url: string): Promise<string> {
+  const { Url } = await import('@rsdoctor/utils/common');
+  if (Url.isUrl(url)) {
+    const resp = await fetch(url);
+    return resp.text();
+  }
+  // json string
+  return url;
+}
+
+export async function parseManifest(
```

Comment:
Where do we use this method?

Reply:
It is used for parsing manifest stats of rsdoctor.
Pasted from [rsdoctor](https://github.com/web-infra-dev/rsdoctor/blob/da541bee23bf339132a4ce3c67a92ffbd1737986/packages/components/src/utils/request.ts#L43) and will be used in the next PR.
For now it just a placeholder.

Reply:
1. Maybe we should discuss with Rsdoctor to get a public API for this.
2. So why this placeholder is added to this PR not next PR.

Reply:
Yep, I remove it from this PR.